test/workspaces.test.ts

```diff
@@ -550,7 +550,6 @@ catalogs:
           output.should.deep.equal({
             'pnpm-workspace.yaml': {
               packages: ['packages/**'],
-              catalog: { 'ncu-test-v2': '2.0.0' },
```

Comment:
this shouldn't happen because `catalog` wasn't defined in the config before so it shouldn't be created by ncu